LICENSE.md

```diff
@@ -16,23 +16,23 @@ Version 2.0.2
 
 **Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
 
-**API**: Software intermediary for interaction with the Mod.
+**API**: Software which is part of the Mod that allows other software to interact with the Mod.
```

Comment:
Still falls into the definition of a "Software intermediary for interaction". This definition covers **anything** that acts as an intermediary. Your changes only covers something that is part of the mod. What if I use an API provided by a modding library, like the CoFHCore, to allow people to interact with a Mod?